R/loadReg2.R

```diff
@@ -444,14 +463,17 @@ predictSolute.loadReg2 <- function(
       }
     }
   }
-  
   # Add dates if requested
   if(date) {
     if(!is.data.frame(preds)) {
       preds <- data.frame(fit=preds)
     }
     # prepend the date column
-    preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #predLoad returns the dates, don't need to get them from metadata
```

Comment:
Hmm, I see that, although `predLoad` and `predConc` both always return data frames containing the dates, so I wasn't sure why they would ever need to be reattached.